### PR TITLE
FIX: Change to interface workdir within Interface.run() instead Node

### PR DESCRIPTION
**Pull request by oesteban**
Addresses #2380 and https://github.com/nipy/nipype/pull/2368#discussion_r160794434

### CODE REVIEW

**Comment by oesteban** [comment]
I've added many reviewers just to make sure we are not missing out any details.

File: nipype/interfaces/base/core.py [hunk 1]
```diff
@@ -471,7 +480,8 @@ def run(self, **inputs):
             env['DISPLAY'] = config.get_display()
 
         runtime = Bunch(
-            cwd=os.getcwd(),
+            cwd=cwd,
+            prevcwd=syscwd,
```
Comment (djarecka) [inline nipype/interfaces/base/core.py]: I couldn't find a place where this is being used

Comment (oesteban) [reply nipype/interfaces/base/core.py]: Oh, forgot to add it to the report.rst

**Comment by oesteban** [comment]
Removed the CHANGES entry:

```
* FIX: Change to interface workdir within Interface.run() instead Node (https://github.com/nipy/nipype/pull/2384)
```

**Pull request merged**